pkg/edged/edged.go

```diff
@@ -113,6 +114,7 @@ type edged struct {
 	rootDirectory             string
 	gpuPluginEnabled          bool
 	version                   string
+	podReadyLock              sync.Mutex
```

Comment:
comment please.

Reply:
If `podReadyLock` is only used for protecting `initPodReady` flag, why not put them into a another struct?

Reply:
moved the flag and the associated lock to a separate struct. embedded this struct as a member variable of edged struct.